debugger/src/main/scala-2.12/org.apache.daffodil.debugger.dap/DAPodil.scala

```diff
@@ -0,0 +1,799 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I'm concerned with how much duplication there is in the scala-2.12, scala-2.13, and scala-3 directories. It looks like there is a lot more than just API shims. There are a lot of logic in these files that feels like could make maintencance very difficult, since any change made in one needs to be duplicated in another.

Feels like these that actual logic could be split out into shared classes so these are basically just shim layers.

Reply:
I think I addressed this in my latest commit. Let me know if it still needs touched up.